# SHA1:d3bcea2bc0b434ee25d52443894a038d22775170
#
# This file is autogenerated by pip-compile-multi
# To update, run:
#
#    pip-compile-multi
#
coverage[toml]==7.4.2
    # via pytest-cov
exceptiongroup==1.2.0
    # via pytest
execnet==2.0.2
    # via pytest-xdist
iniconfig==2.0.0
    # via pytest
numpy==1.26.4
    # via
    #   pandas
    #   pyarrow
packaging==23.2
    # via pytest
pandas==2.2.0
    # via -r basetest.in
pluggy==1.4.0
    # via pytest
psutil==5.9.8
    # via -r basetest.in
pyarrow==15.0.0
    # via -r basetest.in
pytest==8.0.1
    # via
    #   -r basetest.in
    #   pytest-cov
    #   pytest-xdist
pytest-cov==4.1.0
    # via -r basetest.in
pytest-xdist==3.5.0
    # via -r basetest.in
python-dateutil==2.8.2
    # via pandas
pytz==2024.1
    # via pandas
six==1.16.0
    # via python-dateutil
tomli==2.0.1
    # via
    #   coverage
    #   pytest
tzdata==2024.1
    # via pandas
